### PR TITLE
travis: remove old IRC server

### PR DESCRIPTION
**Pull request by MrAnno**

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Review by OverOrion — APPROVED** [review]

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: .travis.yml
Comment (Kokan) [inline .travis.yml:84]: [optional] I think the gitter link can be removed as well. Nobody noticed the lack of notification there. Also majority of the CI is done in github action, so the failure or success of ARM job not that relevant either.

**Review by Kokan — APPROVED** [review]

**Review by szemere — APPROVED** [review]
Leaving time for @Kokan's optional comment. I checked the URL, at least it is working.

File: .travis.yml
Comment (MrAnno) [reply .travis.yml:84]: I opened this one for testing purposes (we had to reregister Travis CI for the syslog-ng repo), but @OverOrion reviewed it, so I thought "let's do it then" :) I don't want to dismiss the 2 approvals on this PR, but I support the idea of removing the Gitter notification, if someone wants to do it.

**Pull request merged**
